recipes/libglvnd/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+from conans import Meson, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import functools
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+class LibGlvndConan(ConanFile):
+    name = "libglvnd"
```

Comment:
As a note: My understanding is, that this might be the better replacement for all the places we currently use `opengl/system`, but this might break compatibility for older systems with older GPU drivers and in general GPU drivers that aren't supported by glvnd.

Reply:
well, some of the distros install glvnd as system opengl too (https://ubuntu.pkgs.org/22.04/ubuntu-main-amd64/libgl-dev_1.4.0-1_amd64.deb.html), so mesa-opengl/system is probably not the good name either.